setup.py

```diff
@@ -4,8 +4,8 @@
     long_description = fh.read()
 
 install_requires=["hyperspy",
-                  "scikit-image>=0.15, <=0.16",
-                  "scikit-learn>=0.21",
+                  "scikit-image>=0.17.2",
+                  "scikit-learn>=0.23.2",
```

Comment:
You probably want to leave these to be the earliest version that still works. Will ParticleSpy still work with earlier versions? Perhaps scikit-image needs to be 0.17 but scikit-learn could probably still be 0.21.

This gives instructions for installation, so we want the instructions to include as many versions as possible to make us compatible with as many installations as possible. i.e. if another package requires scikit-learn < 0.22, we want to avoid a clash with this and so allow installation of 0.21. If the package is not yet installed, the installer will pick the most recent package that matches specifications.

Reply:
Okay will do, I had changed it to test the newest versions, I'll go back and check for the earliest version usable